spirv_glsl.cpp

```diff
@@ -5847,6 +5847,34 @@ bool CompilerGLSL::emit_complex_bitcast(uint32_t result_type, uint32_t id, uint3
 	return true;
 }
 
+static bool check_forward_limit_exceeded(uint32_t op0, uint32_t op1)
```

Comment:
This is not thread safe and way too hacky. I'll try to see how to best solve this.

Reply:
Hmm, I think it would be possible to avoid using static and use context dependent variables (like, add them to the CompilerGLSL class), but I agree this is a hacky way.

Reply:
I think we can use the expression use counter system somehow.